react/week4/github-user-app/src/SearchUsers/SearchUsers.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useContext } from "react";
+import AllUsers from "../AllUsers";
```

Comment:
```suggestion
import Users from "../AllUsers";
```

Putting all in front of Users does not really add any information.